### PR TITLE
feat: add search notes functionality and toasts

### PR DESCRIPTION
**Pull request by anik31**
## Added - 
- toasts
- search notes
- fix some overflow issues

## Files to check
- src/components/Navbar/Navbar.jsx
- src/context/archive-context.js
- src/context/auth-context.js
- src/context/note-context.js
- src/context/trash-context.js
- src/pages/private/notes/Notes.jsx
- src/reducer/note-reducer.js
- src/utils/getFilteredNotes.js
 
 _PS - Ignore other files._

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,10 +13,10 @@ function App() {
   return (
     <div className="App">
       {pathname!=="/" && <Navbar/>}
+        <ToastContainer theme="light" autoClose={2000} position="top-right"/>
```
Comment (sainath7878) [inline src/App.js:16]: Good that you've added toast in your app

File: src/utils/getFilteredNotes.js [hunk 2]
```diff
@@ -21,9 +21,13 @@ export function getFilteredNotes(notes, sortByDate, sortByPriority, label){
...
     }
 
-    if(label.length===0){
-        return notes;
-    }
+    notes = label.length!==0 
+    ? notes.filter(note=>note.labels.some(labelItem=>label.includes(labelItem)))
+    : notes;
+    
+    notes = searchTerm
```
Comment (sainath7878) [inline src/utils/getFilteredNotes.js:28]: Appreciation: Good that you've added search functionality to your application.

**Review by sainath7878 — APPROVED** [review]
Everything's is working as expected. Code is clean and understandable. Great work on the UI too!

File: src/App.js (hunk 1 above)
Comment (anik31) [reply src/App.js:16]: Thanks

File: src/utils/getFilteredNotes.js (hunk 2 above)
Comment (anik31) [reply src/utils/getFilteredNotes.js:28]: Thanks

**Review by partha8 — APPROVED** [review]
Good small PR, nothing new to suggest. Good work, go ahead with the merge.

**Pull request merged**